pkgs/os-specific/darwin/pam-reattach/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub, cmake, clang, openpam }:
+
+stdenv.mkDerivation rec {
+  pname = "pam_reattach";
+  version = "1.3";
+
+  src = fetchFromGitHub {
+    owner = "fabianishere";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1k77kxqszdwgrb50w7algj22pb4fy5b9649cjb08zq9fqrzxcbz7";
+  };
+
+  cmakeFlags = [
+    "-DCMAKE_BUILD_TYPE=Release"
+    "-DCMAKE_OSX_ARCHITECTURES=${
+      if stdenv.hostPlatform.system == "x86_64-darwin" then
+        "x86_64"
+      else
+        "arm64"
+    }"
+    "-DENABLE_CLI=ON"
+  ];
+  enableParallelBuilding = true;
+
+  nativeBuildInputs = [ cmake clang openpam ];
```

Comment:
Isn't openpam not a library and goes to buildInputs?

Reply:
Not sure about that one. Can you please rephrase this without double negative?😬😃
Are you suggesting it's not necessary during build- but just runtime?🤔

Reply:
Okay, tried it. Works. Good catch.